pvlib/shading.py

```diff
@@ -232,3 +232,64 @@ def sky_diffuse_passias(masking_angle):
        Available at https://www.nrel.gov/docs/fy18osti/67399.pdf
     """
     return 1 - cosd(masking_angle/2)**2
+
+
+def projected_solar_zenith_angle(surface_tilt, surface_azimuth,
+                                 solar_apparent_elevation, solar_azimuth):
+    r"""
+    Calculate projected solar zenith angle in degrees.
+
+    This is common in track and shadow computation [1]_ [2]_ [3]_.
+
+    Parameters
+    ----------
+    surface_tilt : numeric
+        Axis tilt angle in degrees. From horizontal plane to array plane.
+    surface_azimuth : numeric
+        Axis azimuth angle in degrees.
+        North = 0°; East = 90°; South = 180°; West = 270°
+    solar_apparent_elevation : numeric
+        Sun's apparent elevation in degrees.
```

Comment:
Let's switch to taking `solar_zenith` as input, and calculate elevation internally.  Also no need for `apparent` in the parameter name (#1403)

Reply:
Was [`variables_style_rules.csv`](https://github.com/pvlib/pvlib-python/blob/4e55d500821847b1876c443cbaef4ecc4fe0b05e/pvlib/data/variables_style_rules.csv#L23C1-L27) updated according to that conversation?
I see

```
surface_tilt;tilt angle of the surface
surface_azimuth;azimuth angle of the surface
solar_zenith;zenith angle of the sun in degrees
apparent_zenith;refraction-corrected solar zenith angle in degrees
solar_azimuth;azimuth angle of the sun in degrees East of North
```
At least `solar_zenith` suggests me it's true solar zenith, since apparent is mentioned right after and it does not have "refraction-corrected" in the definition.
I've kept "apparent" in the parameter description just in case.